pco_python/test/test_standalone.py

```diff
@@ -92,13 +113,52 @@ def test_compression_options():
 
   # this is mostly just to check that there is no error, but these settings
   # should give worse compression than the defaults
-  assert len(standalone.simple_compress(
-    data,
-    ChunkConfig(
-      compression_level=0,
-      delta_encoding_order=1,
-      int_mult_spec='disabled',
-      float_mult_spec='DISABLED',
-      paging_spec=PagingSpec.equal_pages_up_to(77),
-    )
-  )) > default_size
+  assert (len(
+      standalone.simple_compress(
+          data,
+          ChunkConfig(
+              compression_level=0,
+              delta_encoding_order=1,
+              int_mult_spec=IntMultSpec.disabled(),
+              float_mult_spec=FloatMultSpec.disabled(),
+              float_quant_spec=FloatQuantSpec.disabled(),
+              paging_spec=PagingSpec.equal_pages_up_to(77),
+          ),
+      )) > default_size)
+
+
+@pytest.mark.parametrize("int_mult_spec", all_int_mult_specs)
```

Comment:
Hmm, I don't think this really tests that the right configuration is being used. I think instead we should just consider a distribution where the mode is definitely beneficial and confirm we get better sizes using `enabled()` or `provided(correct_answer)`.

To get such a distribution for mult modes, you could set `scale=777` (probably anything >500 would work) in the `np.random.normal` call.

Really the best thing would be based on chunk metadata instead of compressed size, but the Python API doesn't support inspecting chunk metadata yet.

Reply:
The tests were primarily added to check that all the configs work in the sense that they don't crash or return empty. I've changed the tests to make the intent more clear.

If you'd like the tests to check that the different options work as expected for specific distributions, perhaps you can fix that? I'm not sure how to do it properly.

Reply:
Ok, this seems fine for now